iggy-enrich-demo/README.md

```diff
@@ -0,0 +1,32 @@
+# iggy-enrich-demo
+
+This repo contains code for a simple demo of using Iggy feature enrichment.
+We will enrich some vacation rental data, explore it and then apply the dataset to two tasks:
+- vacation rental recommendation
+- vacation rental ratings prediction
+
+## Getting started
+
+
+- From the root directory of the repo, set up virtual environment and install dependencies, e.g.:
```

Comment:
root directory of the _demo_ (or just add a `cd iggy-enrich-demo` to the code block?)